cypress/page/payment-step.page.ts

```diff
@@ -0,0 +1,33 @@
+class PaymentStep {
+
+    private paymentMethod: string
+    private confirmOrder: string
+    private signOutAfterPayment: string
+    private confirmationMessage: string
+
+    constructor() {
+        this.paymentMethod = ".bankwire"
+        this.confirmOrder = ".button[type='submit']"
+        this.signOutAfterPayment = ".logout[title='Log me out']"
+        this.confirmationMessage = ".cheque-indent"
+
+    }
+
+    public selectPaymentMethod(): void {
+        cy.get(this.paymentMethod).click()
+    }
+
+    public confirmTheOrder(): void {
+        cy.get(this.confirmOrder).contains("I confirm my order").click()
+    }
+
+    public confirmationPaymentMessage(): void {
+        cy.get(this.confirmationMessage).contains("Your order on My Store is complete.")
```

Comment:
receive the expected message as input param of the method, also rename the method to be validateConfirmationPaymentMessage or something like that, in order to indicate we are asserting or validating things here

Reply:
Got it! Validate and verify, the QA's verbs. At this time, Verify the "acceptance criteria"